src/core/tests/Test_PatchInstaller.py

```diff
@@ -55,6 +58,30 @@ def test_yum_install_success(self):
         runtime = RuntimeCompositor(argument_composer.get_composed_arguments(), True, Constants.YUM)
         # Path change
         runtime.set_legacy_test_type('SuccessInstallPath')
+        # As all the packages should get installed using batch patching, get_remaining_packages_to_install should return 0 packages
+        backup_get_remaining_packages_to_install = runtime.patch_installer.get_remaining_packages_to_install
+        runtime.patch_installer.get_remaining_packages_to_install = self.mock_get_remaining_packages_to_install
+        installed_update_count, update_run_successful, maintenance_window_exceeded = runtime.patch_installer.install_updates(runtime.maintenance_window, runtime.package_manager, simulate=True)
+        runtime.patch_installer.get_remaining_packages_to_install = backup_get_remaining_packages_to_install
```

Comment:
this should be at the end after Assert phase. Please go over all the tests and change accordingly.

Reply:
Done